### ISSUE TITLE
Merge runs and set colors for plot_train.py

### ISSUE THREAD

**Issue by Matteo-Salis** [ecmwf/WeatherGenerator#2455]
When continuing training, it will be useful to print all loss plots from `plot_train.py` by merging the run_ids of the original and resumed runs, and keep a single unique label for them.

It will also be very useful to be able to specify (optionally) colors. This will allow us to keep consistent colors between fasteval plots and plot_train.py ones.

### Area

- [ ] datasets, data readers, data preparation and transfer
- [ ] model
- [ ] science
- [x] infrastructure and engineering
- [x] evaluation, export and visualization
- [ ] documentation
- [ ] performance

**Comment by clessig** [comment]
An alternative to merging runs is to have the ability to assign specific colours to runs. This is also useful to have `red-dashed` for pre-training and `red-solid` for fine-tuning.
